### ISSUE TITLE
typings for angular ngMocks (angular 1.x)

### ISSUE THREAD

**Issue by fade2g** [typings/registry#166]
In order use angular 1.5 with typescript and be able to write unit tests in typescript as well, it would be great to have typings for ngMock.

There seam to be [ngMocks typings for tsd](https://raw.githubusercontent.com/DefinitelyTyped/DefinitelyTyped/master/angularjs/angular-mocks.d.ts) but I have no idea, how to use them with typings.

**Comment by johnnyreilly** [comment]
I think this is what you need:

```
typings search angularjs/angular-mocks --ambient

Showing 1 results...

NAME                    SOURCE HOMEPAGE             DESCRIPTION UPDATED                  VERSIONS
angularjs/angular-mocks dt     http://angularjs.org             2016-02-15T16:20:30.000Z 27
```

**Comment by fade2g** [comment]
Thanks, didn't know about prefixing "angularjs/" while searching with variations of angular-mocks. But: Installing fails, as it looks like the file is not found:

`typings install angularjs/angular-mocks --ambient -save`

results in (shortened some paths in copied code)
```
? Found angularjs/angular-mocks typings for DefinitelyTyped. Continue? Yes
Installing angularjs/angular-mocks@0.0.0+20160215162030 (DefinitelyTyped)...

typings ERR! message ENOENT: no such file or directory, open '....typings\browser\ambient\angularjs\angular-mocks\angularjs\angular-mocks.d.ts'

typings ERR! stack Error: ENOENT: no such file or directory, open '....typings\browser\ambient\angularjs\angular-mocks\angularjs\angular-mocks.d.ts'
typings ERR! stack     at Error (native)

typings ERR! cwd C:\Users\...
typings ERR! system Windows_NT 6.1.7601
typings ERR! command "C:\\Program Files\\nodejs\\node.exe" "...typings\\dist\\bin\\typings-install.js" "angularjs/angular-mocks" "--ambient" "-save"
typings ERR! node -v v4.3.1
typings ERR! typings -v 0.6.9
typings ERR! code ENOENT

typings ERR! If you need help, you may report this error at:
typings ERR!   <https://github.com/typings/typings/issues>
```

**Comment by unional** [comment]
Very likely it is a bug. I can help to take a look into it

**Comment by unional** [comment]
dup of https://github.com/typings/typings/issues/275

**Comment by fade2g** [comment]
Thanks for the pointer to the issue. Installation using 
`typings install angularjs/angular-mocks --name angular-mocks --ambient --save` worked.

My manual approach for possible future reference I just had written:
I'm very new to TypeScript and usage of typings. I noticed that there is a difference. sometimes, there is angular being used and sometimes angularjs. I figured out, how to manually add a functioning reference to typings.json

`"angular-mocks": "github:DefinitelyTyped/DefinitelyTyped/angularjs/angular-mocks.d.ts#4de74cb527395c13ba20b438c3a7a419ad931f1c"`

I came to that solution after recognising that the installed anguar typings (installed with
`typings install angular --ambient --save` where resolved to a quite old definition github:DefinitelyTyped/DefinitelyTyped/angularjs/legacy/angular-1.2.d.ts#17d8aadb533b1c3ae462f8bcfcfa758bb38e55c1` (I noticed, as IComponent interface was missing).

Browsing the git repo I figured out that the angular typings should be `github:DefinitelyTyped/DefinitelyTyped/angularjs/angular.d.ts#49a247335032771e77146ba4c328bb506c0eff78` and used similarily the reference above to.
